web/src/app/administration/components/classification-details/classification-details.component.ts

```diff
@@ -80,7 +78,9 @@ export class ClassificationDetailsComponent implements OnInit, OnDestroy {
     this.action$.pipe(takeUntil(this.destroy$)).subscribe(data => {
       this.action = data;
       if (this.action === ACTION.CREATE) {
-        this.initClassificationOnCreation();
+        this.selectedClassification$.pipe(take(1)).subscribe(initialClassification => {
```

Comment:
I'm missing a `takeUntil(this.destroy$)`

Reply:
this is not needed since there is take(1)